app/controllers/books/comments_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Books::CommentsController < ApplicationController
+  before_action :set_book, only: %i[create destroy]
+  def create
+    if @book.comments.build(content: comment_params[:content], user: current_user).save
```

Comment:
一度に全部やろうとせずに処理を分けた方が可読性が高いかなと思います。

```suggestion
    comment = @book.comments.build(comment_params)
    comment.user = current_user
    if comment.save
```

Reply:
[6ec9400](https://github.com/s-miura7/fjord-books_app-2023/pull/5/commits/6ec940087bddf0db1b89df6932afc3cae9dc99e3) で修正